glean-core/src/metrics/timing_distribution.rs

```diff
@@ -314,6 +314,64 @@ impl TimingDistributionMetric {
         }
     }
 
+    /// Accumulates the provided samples in the metric.
+    ///
+    /// # Arguments
+    ///
+    /// * `samples` - A list of samples recorded by the metric.
+    ///               Samples must be in nanoseconds.
+    /// ## Notes
+    ///
+    /// Reports an [`ErrorType::InvalidOverflow`] error for samples that
+    /// are longer than `MAX_SAMPLE_TIME`.
+    /// Samples must be in nanoseconds, based on time from a monotonic clock.
```

Comment:
Do they really have to be based on a monotonic clock? I mean, I guess they have to be nonnegative (because they're `u64`) which requires them to be from an increasing clock? Not sure this phrase is adding much.

Reply:
They should, but I guess we take whatever as long as you promise it's some duration.

Reply:
(will change the sentence)